docs/pages/_document.js

```diff
@@ -7,6 +7,7 @@ import { LANGUAGES_SSR } from 'docs/src/modules/constants';
 import { pathnameToLanguage } from 'docs/src/modules/utils/helpers';
 import createEmotionCache from 'docs/src/createEmotionCache';
 import { getMetaThemeColor } from 'docs/src/modules/brandingTheme';
+import { GlobalStyles } from '@mui/styled-engine';
```

Comment:
The other components in the docs are importing from:

```suggestion
import GlobalStyles from '@mui/material/GlobalStyles';
```